packages/camera-web/test/SimpleCameraHUD/SimpleCameraHUD.test.tsx

```diff
@@ -38,51 +40,33 @@ describe('SimpleCameraHUD component', () => {
     unmount();
   });
 
-  it('should display a take picture button that calls the handle.takePicture method', () => {
-    const handle = { takePicture: jest.fn() };
-    const { unmount } = render(<SimpleCameraHUD handle={handle} />);
-
-    fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-    expect(handle.takePicture).toHaveBeenCalled();
-    unmount();
-  });
-
-  it('should not throw any error when no handle is provided', () => {
-    expect(() => {
-      const { unmount, rerender } = render(<SimpleCameraHUD />);
-      fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-
-      rerender(<SimpleCameraHUD handle={{}} />);
-      fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-      unmount();
-    }).not.toThrow();
-  });
```

Comment:
You should keep this test! A test that verifies that the test button is indeed in the virtual DOM. You can do it like that :

```typescript jsx
it('should display a take picture button', () => {
  const handle = { takePicture: jest.fn() };
  const { unmount } = render(<SimpleCameraHUD />);

  expect(TakePictureButtonMock).toHaveBeenCalled();
  expect(screen.queryByTestId(TAKE_PICTURE_MOCK_BTN_TEST_ID)).toBeDefined();
  unmount();
});
```